website/pages/playground/index.html

```diff
@@ -246,41 +311,41 @@
         <textarea class="loading" id="output-editor"></textarea>
       </div>
     </div>
-  </section>
+  </div>
 
-  <section>
+  <div>
     <details>
       <summary class="options-summary">Options</summary>
 
       <div class="options-container">
         <div class="options">
-          <label>--print-width <input type="number" value="80" min="0" id="printWidth"></input> </label>
-          <label>--tab-width <input type="number" min="0" value="2" id="tabWidth"></input> </label>
+          <label>--print-width <input type="number" value="80" min="0" id="printWidth"></label>
+          <label>--tab-width <input type="number" min="0" value="2" id="tabWidth"></label>
         </div>
         <div class="options">
-          <label><input type="checkbox" id="useTabs"></input> --use-tabs</label>
-          <label><input type="checkbox" data-inverted id="semi"></input> --no-semi</label>
-          <label><input type="checkbox" id="singleQuote"></input> --single-quote</label>
-          <label><input type="checkbox" data-inverted id="bracketSpacing"></input> --no-bracket-spacing</label>
-          <label><input type="checkbox" id="jsxBracketSameLine"></input> --jsx-bracket-same-line</label>
+          <label><input type="checkbox" id="useTabs"> --use-tabs</label>
+          <label><input type="checkbox" data-inverted id="semi"> --no-semi</label>
+          <label><input type="checkbox" id="singleQuote"> --single-quote</label>
+          <label><input type="checkbox" data-inverted id="bracketSpacing"> --no-bracket-spacing</label>
+          <label><input type="checkbox" id="jsxBracketSameLine"> --jsx-bracket-same-line</label>
         </div>
         <div class="options last">
           <label>--trailing-comma <select id="trailingComma"><option value="none">none</option><option value="es5">es5</option><option value="all">all</option></select></label>
           <label>--parser <select id="parser"><option value="babylon">babylon</option><option value="flow">flow</option><option value="typescript">typescript</option><option value="postcss">postcss</option><option value="json">json</option><option value="graphql">graphql</option></select></label>
           <span style="flex: 0.3"></span>
-          <label><input type="checkbox" id="ast"></input> show AST (debug)</label>
-          <label><input type="checkbox" id="doc"></input> show doc (debug)</label>
+          <label><input type="checkbox" id="ast"> show AST (debug)</label>
+          <label><input type="checkbox" id="doc"> show doc (debug)</label>
         </div>
       </div>
     </details>
-  </section>
+  </div>
 
   <footer>
     <p class="version-link">
       <a href="https://github.com/prettier/prettier">
         prettier version
         <span class="version"></span>
-        (master)
+        / master
```

Comment:
The playground itself is redeployed but the version of prettier is not automatically rebuilt

Reply:
Is there a disadvantage to let the playground always use latest master? If so, I'll look into updating website/deploy.sh to also build Prettier.

Reply:
Not that I can think of...

Reply:
Yes, it's confusing for people to see master on the playground where they can't actually use it. We should have the last release by default and if we can manage, then having an option to display other versions.

We're going to do less drastic changes going forward so it may not be such a big deal anymore though.

Reply:
Ok, I'll leave it to use the latest release, then.